utils/db.go

```diff
@@ -80,3 +81,23 @@ func (h DbAndSchema) GetViewDirective() string {
 		return ""
 	}
 }
+func ToSQL(query *gorm.DB) string {
```

Comment:
a suggestion here is to add a unit test for this function

Reply:
done